src/_tests_/components/forecast-details.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, cleanup} from '@testing-library/react';
+import ForecastDetails from '../../components/forecast-details';
+
+afterEach(cleanup);
+
+describe('ForecastDetails', () => {
+  const mockForecast = {
+    date: 1525046400000,
+    temperature: {
+      max: 16,
+      min: 5, 
+    },
+    humidity: 30,
+    wind: {
+      speed: 13,
+    },
+  };
+
+  it('renders correctly', () => {
+    const { asFragment } = render(<ForecastDetails forecasts={mockForecast} />);
+    expect(asFragment).toMatchSnapshot();
+  });
+  it('renders the correct amount of props', ()=> {
+    const { getByText, getByTestId } = render(
+      <ForecastDetails forecasts={mockForecast} />
+    );
+    //const { getByTestId } = render(
+     // <ForecastDetails forcasts={mockForecast}/>
```

Comment:
delete if not needed. There is a typo in the prop forcasts should be forecasts, that could have been a reason it wasn't running.